css/cssom/cssstyledeclaration-setter-form-controls.html WPT test is failing in WebKit

cdumez opened the pull request:
#### fc87a93fd5ce158d320f36a8b47665fbab9bee4a
<pre>
css/cssom/cssstyledeclaration-setter-form-controls.html WPT test is failing in WebKit
<a href="https://bugs.webkit.org/show_bug.cgi?id=107380">https://bugs.webkit.org/show_bug.cgi?id=107380</a>
&lt;rdar://problem/34281872&gt;

Reviewed by Darin Adler.

We are failing this test because we are the only browser that adds intrinsics margins
to form controls, now that Blink stopped doing this in:
- <a href="http://code.google.com/p/chromium/issues/detail?id=128306">http://code.google.com/p/chromium/issues/detail?id=128306</a>

Being the only browser adding such margins is an interoperability issue so we should
align. I am however making the change behind a linked-on-after check to avoid breaking
apps that may rely on this.

* LayoutTests/fast/flexbox/clear-overflow-before-scroll-update-expected.txt:
* LayoutTests/fast/forms/auto-fill-button/input-credit-card-auto-fill-button-expected.txt:
* LayoutTests/fast/forms/button-with-float-expected.html:
* LayoutTests/fast/forms/hidpi-textfield-background-bleeding.html:
* LayoutTests/fast/forms/listbox-clip-expected.txt:
* LayoutTests/fast/forms/option-mouseevents-expected.txt:
* LayoutTests/fast/html/text-field-input-types-expected.txt:
* LayoutTests/fast/images/mac/image-controls-sticky-scrolling-expected.txt:
* LayoutTests/fast/multicol/flexbox-rows-expected.html:
* LayoutTests/fast/multicol/flexbox-rows.html:
* LayoutTests/fast/replaced/pdf-as-embed-with-no-mime-type-is-not-blank.html:
* LayoutTests/fast/scrolling/latching/scroll-select-bottom-test-expected.txt:
* LayoutTests/fast/scrolling/latching/scroll-select-latched-mainframe-expected.txt:
* LayoutTests/fast/scrolling/latching/scroll-select-latched-select-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/scroll-target-margin-005-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/cssstyledeclaration-setter-form-controls-expected.txt:
* LayoutTests/platform/mac-wk1/fast/dom/collection-iterators-expected.txt:
* LayoutTests/platform/mac-wk2/fast/forms/date/date-input-rendering-basic-expected.txt:
* LayoutTests/platform/mac-wk2/fast/forms/date/date-pseudo-elements-expected.txt:
* LayoutTests/platform/mac-wk2/fast/forms/time/time-input-rendering-basic-expected.txt:
* LayoutTests/platform/mac/fast/attachment/attachment-image-controls-basic-expected.txt:
* LayoutTests/platform/mac/fast/block/float/float-avoidance-expected.txt:
* LayoutTests/platform/mac/fast/block/float/overhanging-tall-block-expected.txt:
* LayoutTests/platform/mac/fast/block/margin-collapse/103-expected.txt:
* LayoutTests/platform/mac/fast/block/positioning/inline-block-relposition-expected.txt:
* LayoutTests/platform/mac/fast/css/continuationCrash-expected.txt:
* LayoutTests/platform/mac/fast/css/focus-ring-exists-for-search-field-expected.txt:
* LayoutTests/platform/mac/fast/css/line-height-expected.txt:
* LayoutTests/platform/mac/fast/css/margin-top-bottom-dynamic-expected.txt:
* LayoutTests/platform/mac/fast/css/rtl-ordering-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-input-expected.txt:
* LayoutTests/platform/mac/fast/css/text-transform-select-expected.txt:
* LayoutTests/platform/mac/fast/dom/HTMLTableColElement/resize-table-using-col-width-expected.txt:
* LayoutTests/platform/mac/fast/dom/HTMLTextAreaElement/reset-textarea-expected.txt:
* LayoutTests/platform/mac/fast/dynamic/008-expected.txt:
* LayoutTests/platform/mac/fast/dynamic/positioned-movement-with-positioned-children-expected.txt:
* LayoutTests/platform/mac/fast/events/context-no-deselect-expected.txt:
* LayoutTests/platform/mac/fast/events/shadow-event-path-2-expected.txt:
* LayoutTests/platform/mac/fast/forms/001-expected.txt:
* LayoutTests/platform/mac/fast/forms/003-expected.txt:
* LayoutTests/platform/mac/fast/forms/004-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/hide-auto-fill-strong-password-viewable-treatment-when-form-is-reset-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-auto-fill-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-contacts-auto-fill-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-strong-password-auto-fill-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-strong-password-viewable-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-buttons-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-inputs-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-selects-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-textareas-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-textareas-quirks-expected.txt:
* LayoutTests/platform/mac/fast/forms/blankbuttons-expected.txt:
* LayoutTests/platform/mac/fast/forms/box-shadow-override-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-cannot-be-nested-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-generated-content-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-inner-block-reuse-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-positioned-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-sizes-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-style-color-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-table-styles-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-text-transform-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-white-space-expected.txt:
* LayoutTests/platform/mac/fast/forms/control-clip-overflow-expected.txt:
* LayoutTests/platform/mac/fast/forms/control-restrict-line-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/datalist/datalist-searchinput-appearance-expected.txt:
* LayoutTests/platform/mac/fast/forms/datalist/datalist-textinput-appearance-expected.txt:
* LayoutTests/platform/mac/fast/forms/disabled-select-change-index-expected.txt:
* LayoutTests/platform/mac/fast/forms/encoding-test-expected.txt:
* LayoutTests/platform/mac/fast/forms/fieldset-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/file/file-input-direction-expected.txt:
* LayoutTests/platform/mac/fast/forms/file/file-input-disabled-expected.txt:
* LayoutTests/platform/mac/fast/forms/file/input-file-re-render-expected.txt:
* LayoutTests/platform/mac/fast/forms/floating-textfield-relayout-expected.txt:
* LayoutTests/platform/mac/fast/forms/form-element-geometry-expected.txt:
* LayoutTests/platform/mac/fast/forms/formmove3-expected.txt:
* LayoutTests/platform/mac/fast/forms/hidden-listbox-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-bkcolor-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-default-bkcolor-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-disabled-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-focus-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-preventDefault-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-readonly-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-selection-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-spinbutton-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-spinbutton-up-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-visibility-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-baseline-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-button-sizes-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-disabled-color-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-double-click-selection-gap-bug-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-field-text-truncated-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-first-letter-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-placeholder-visibility-1-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-placeholder-visibility-3-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-readonly-autoscroll-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-readonly-dimmed-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-readonly-empty-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-spaces-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-table-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-click-inside-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-click-outside-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-double-click-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-drag-down-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-option-delete-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-scroll-left-on-blur-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-self-emptying-click-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-word-wrap-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-type-text-min-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-value-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-bidi-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-hit-test-zoomed-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-scrollbar-incremental-load-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-width-change-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-clip-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-deselect-update-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-narrow-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-option-wrap-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-restrict-line-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-separator-painting-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-style-color-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-width-change-expected.txt:
* LayoutTests/platform/mac/fast/forms/minWidthPercent-expected.txt:
* LayoutTests/platform/mac/fast/forms/number/number-appearance-rtl-expected.txt:
* LayoutTests/platform/mac/fast/forms/number/number-appearance-spinbutton-disabled-readonly-expected.txt:
* LayoutTests/platform/mac/fast/forms/number/number-appearance-spinbutton-layer-expected.txt:
* LayoutTests/platform/mac/fast/forms/option-script-expected.txt:
* LayoutTests/platform/mac/fast/forms/option-strip-whitespace-expected.txt:
* LayoutTests/platform/mac/fast/forms/option-text-clip-expected.txt:
* LayoutTests/platform/mac/fast/forms/placeholder-position-expected.txt:
* LayoutTests/platform/mac/fast/forms/placeholder-pseudo-style-expected.txt:
* LayoutTests/platform/mac/fast/forms/plaintext-mode-2-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-cancel-button-style-sharing-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-display-none-cancel-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-input-rtl-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-rtl-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-styled-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-vertical-alignment-expected.txt:
* LayoutTests/platform/mac/fast/forms/search/search-padding-cancel-results-buttons-expected.txt:
* LayoutTests/platform/mac/fast/forms/search/search-size-with-decorations-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-background-none-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-baseline-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-block-background-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-change-listbox-size-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-change-listbox-to-popup-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-change-popup-to-listbox-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-dirty-parent-pref-widths-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-disabled-appearance-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-element-focus-ring-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-empty-option-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-initial-position-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-item-background-clip-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-list-box-with-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-listbox-multiple-no-focusring-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-non-native-rendering-direction-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-overflow-scroll-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-overflow-scroll-inherited-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-selected-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-style-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-visual-hebrew-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-writing-direction-natural-expected.txt:
* LayoutTests/platform/mac/fast/forms/select/optgroup-rendering-expected.txt:
* LayoutTests/platform/mac/fast/forms/selectlist-minsize-expected.txt:
* LayoutTests/platform/mac/fast/forms/stuff-on-my-optgroup-expected.txt:
* LayoutTests/platform/mac/fast/forms/tabbing-input-iframe-expected.txt:
* LayoutTests/platform/mac/fast/forms/targeted-frame-submission-expected.txt:
* LayoutTests/platform/mac/fast/forms/textAreaLineHeight-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-placeholder-pseudo-style-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-placeholder-visibility-1-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-placeholder-visibility-2-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-scrollbar-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-scrolled-type-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-setinnerhtml-expected.txt:
* LayoutTests/platform/mac/fast/forms/textfield-focus-ring-expected.txt:
* LayoutTests/platform/mac/fast/forms/textfield-outline-expected.txt:
* LayoutTests/platform/mac/fast/forms/textfield-overflow-expected.txt:
* LayoutTests/platform/mac/fast/forms/visual-hebrew-text-field-expected.txt:
* LayoutTests/platform/mac/fast/frames/take-focus-from-iframe-expected.txt:
* LayoutTests/platform/mac/fast/hidpi/resize-corner-hidpi-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/mac/fast/invalid/014-expected.txt:
* LayoutTests/platform/mac/fast/lists/dynamic-marker-crash-expected.txt:
* LayoutTests/platform/mac/fast/loader/non-html-load-event-expected.txt:
* LayoutTests/platform/mac/fast/overflow/overflow-x-y-expected.txt:
* LayoutTests/platform/mac/fast/overflow/scroll-nested-positioned-layer-in-overflow-expected.txt:
* LayoutTests/platform/mac/fast/overflow/scrollRevealButton-expected.txt:
* LayoutTests/platform/mac/fast/parser/document-write-option-expected.txt:
* LayoutTests/platform/mac/fast/parser/entity-comment-in-textarea-expected.txt:
* LayoutTests/platform/mac/fast/parser/open-comment-in-textarea-expected.txt:
* LayoutTests/platform/mac/fast/repaint/renderer-destruction-by-invalidateSelection-crash-expected.txt:
* LayoutTests/platform/mac/fast/repaint/search-field-cancel-expected.txt:
* LayoutTests/platform/mac/fast/repaint/select-option-background-color-expected.txt:
* LayoutTests/platform/mac/fast/repaint/subtree-root-skipped-expected.txt:
* LayoutTests/platform/mac/fast/replaced/replaced-breaking-expected.txt:
* LayoutTests/platform/mac/fast/replaced/replaced-breaking-mixture-expected.txt:
* LayoutTests/platform/mac/fast/replaced/three-selects-break-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-button-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-menulist-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-searchfield-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-textarea-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-textfield-expected.txt:
* LayoutTests/platform/mac/fast/selectors/064-expected.txt:
* LayoutTests/platform/mac/fast/spatial-navigation/snav-multiple-select-focusring-expected.txt:
* LayoutTests/platform/mac/fast/table/003-expected.txt:
* LayoutTests/platform/mac/fast/table/append-cells2-expected.txt:
* LayoutTests/platform/mac/fast/table/colspanMinWidth-expected.txt:
* LayoutTests/platform/mac/fast/table/remove-td-display-none-expected.txt:
* LayoutTests/platform/mac/fast/table/spanOverlapRepaint-expected.txt:
* LayoutTests/platform/mac/fast/table/text-field-baseline-expected.txt:
* LayoutTests/platform/mac/fast/text/drawBidiText-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-listbox-atsui-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-listbox-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-menulist-expected.txt:
* LayoutTests/platform/mac/fast/text/international/hindi-spacing-expected.txt:
* LayoutTests/platform/mac/fast/text/international/pop-up-button-text-alignment-and-direction-expected.txt:
* LayoutTests/platform/mac/fast/text/international/rtl-white-space-pre-wrap-expected.txt:
* LayoutTests/platform/mac/fast/text/international/unicode-bidi-plaintext-in-textarea-expected.txt:
* LayoutTests/platform/mac/fast/text/textIteratorNilRenderer-expected.txt:
* LayoutTests/platform/mac/fast/transforms/transformed-focused-text-input-expected.txt:
* Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h:
* Source/WebCore/style/StyleAdjuster.cpp:
(WebCore::Style::Adjuster::adjust const):

Canonical link: <a href="https://commits.webkit.org/253424@main">https://commits.webkit.org/253424@main</a>
</pre>

pxlcoder commented on LayoutTests/fast/forms/option-mouseevents-expected.txt:
```diff
@@ -75,15 +75,15 @@ PASS: event type should be mousedown and is
-PASS: event.offsetX should be 12 and is
-PASS: event.offsetY should be 38 and is
+FAIL: event.offsetX should be 12 but instead is 14
+FAIL: event.offsetY should be 38 but instead is 40
```
Probably want to update the test itself.

cdumez replied:
Fixed.

pxlcoder commented on LayoutTests/fast/forms/option-mouseevents.html:
```diff
@@ -38,8 +38,8 @@
-    shouldBe("event.offsetX", event.offsetX, offsetX);
-    shouldBe("event.offsetY", event.offsetY, offsetY);
+    shouldBe("event.offsetX", event.offsetX, offsetX + 2);
+    shouldBe("event.offsetY", event.offsetY, offsetY + 2);
```
I think it would be better to update the call sites of `mouseeventverify2` rather than have a special `+ 2` here.

cdumez committed:
css/cssom/cssstyledeclaration-setter-form-controls.html WPT test is failing in WebKit
https://bugs.webkit.org/show_bug.cgi?id=107380
<rdar://problem/34281872>

Reviewed by Darin Adler.

We are failing this test because we are the only browser that adds intrinsics margins
to form controls, now that Blink stopped doing this in:
- http://code.google.com/p/chromium/issues/detail?id=128306

Being the only browser adding such margins is an interoperability issue so we should
align. I am however making the change behind a linked-on-after check to avoid breaking
apps that may rely on this.

* LayoutTests/fast/flexbox/clear-overflow-before-scroll-update-expected.txt:
* LayoutTests/fast/forms/auto-fill-button/input-credit-card-auto-fill-button-expected.txt:
* LayoutTests/fast/forms/button-with-float-expected.html:
* LayoutTests/fast/forms/hidpi-textfield-background-bleeding.html:
* LayoutTests/fast/forms/listbox-clip-expected.txt:
* LayoutTests/fast/forms/option-mouseevents-expected.txt:
* LayoutTests/fast/html/text-field-input-types-expected.txt:
* LayoutTests/fast/images/mac/image-controls-sticky-scrolling-expected.txt:
* LayoutTests/fast/multicol/flexbox-rows-expected.html:
* LayoutTests/fast/multicol/flexbox-rows.html:
* LayoutTests/fast/replaced/pdf-as-embed-with-no-mime-type-is-not-blank.html:
* LayoutTests/fast/scrolling/latching/scroll-select-bottom-test-expected.txt:
* LayoutTests/fast/scrolling/latching/scroll-select-latched-mainframe-expected.txt:
* LayoutTests/fast/scrolling/latching/scroll-select-latched-select-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/scroll-target-margin-005-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/cssstyledeclaration-setter-form-controls-expected.txt:
* LayoutTests/platform/mac-wk1/fast/dom/collection-iterators-expected.txt:
* LayoutTests/platform/mac-wk2/fast/forms/date/date-input-rendering-basic-expected.txt:
* LayoutTests/platform/mac-wk2/fast/forms/date/date-pseudo-elements-expected.txt:
* LayoutTests/platform/mac-wk2/fast/forms/time/time-input-rendering-basic-expected.txt:
* LayoutTests/platform/mac/fast/attachment/attachment-image-controls-basic-expected.txt:
* LayoutTests/platform/mac/fast/block/float/float-avoidance-expected.txt:
* LayoutTests/platform/mac/fast/block/float/overhanging-tall-block-expected.txt:
* LayoutTests/platform/mac/fast/block/margin-collapse/103-expected.txt:
* LayoutTests/platform/mac/fast/block/positioning/inline-block-relposition-expected.txt:
* LayoutTests/platform/mac/fast/css/continuationCrash-expected.txt:
* LayoutTests/platform/mac/fast/css/focus-ring-exists-for-search-field-expected.txt:
* LayoutTests/platform/mac/fast/css/line-height-expected.txt:
* LayoutTests/platform/mac/fast/css/margin-top-bottom-dynamic-expected.txt:
* LayoutTests/platform/mac/fast/css/rtl-ordering-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-input-expected.txt:
* LayoutTests/platform/mac/fast/css/text-transform-select-expected.txt:
* LayoutTests/platform/mac/fast/dom/HTMLTableColElement/resize-table-using-col-width-expected.txt:
* LayoutTests/platform/mac/fast/dom/HTMLTextAreaElement/reset-textarea-expected.txt:
* LayoutTests/platform/mac/fast/dynamic/008-expected.txt:
* LayoutTests/platform/mac/fast/dynamic/positioned-movement-with-positioned-children-expected.txt:
* LayoutTests/platform/mac/fast/events/context-no-deselect-expected.txt:
* LayoutTests/platform/mac/fast/events/shadow-event-path-2-expected.txt:
* LayoutTests/platform/mac/fast/forms/001-expected.txt:
* LayoutTests/platform/mac/fast/forms/003-expected.txt:
* LayoutTests/platform/mac/fast/forms/004-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/hide-auto-fill-strong-password-viewable-treatment-when-form-is-reset-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-auto-fill-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-contacts-auto-fill-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-strong-password-auto-fill-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/auto-fill-button/input-strong-password-viewable-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-buttons-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-inputs-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-selects-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-textareas-expected.txt:
* LayoutTests/platform/mac/fast/forms/basic-textareas-quirks-expected.txt:
* LayoutTests/platform/mac/fast/forms/blankbuttons-expected.txt:
* LayoutTests/platform/mac/fast/forms/box-shadow-override-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-cannot-be-nested-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-generated-content-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-inner-block-reuse-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-positioned-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-sizes-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-style-color-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-table-styles-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-text-transform-expected.txt:
* LayoutTests/platform/mac/fast/forms/button-white-space-expected.txt:
* LayoutTests/platform/mac/fast/forms/control-clip-overflow-expected.txt:
* LayoutTests/platform/mac/fast/forms/control-restrict-line-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/datalist/datalist-searchinput-appearance-expected.txt:
* LayoutTests/platform/mac/fast/forms/datalist/datalist-textinput-appearance-expected.txt:
* LayoutTests/platform/mac/fast/forms/disabled-select-change-index-expected.txt:
* LayoutTests/platform/mac/fast/forms/encoding-test-expected.txt:
* LayoutTests/platform/mac/fast/forms/fieldset-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/file/file-input-direction-expected.txt:
* LayoutTests/platform/mac/fast/forms/file/file-input-disabled-expected.txt:
* LayoutTests/platform/mac/fast/forms/file/input-file-re-render-expected.txt:
* LayoutTests/platform/mac/fast/forms/floating-textfield-relayout-expected.txt:
* LayoutTests/platform/mac/fast/forms/form-element-geometry-expected.txt:
* LayoutTests/platform/mac/fast/forms/formmove3-expected.txt:
* LayoutTests/platform/mac/fast/forms/hidden-listbox-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-bkcolor-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-default-bkcolor-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-disabled-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-focus-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-preventDefault-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-readonly-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-selection-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-spinbutton-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-spinbutton-up-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-visibility-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-appearance-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-baseline-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-button-sizes-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-disabled-color-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-double-click-selection-gap-bug-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-field-text-truncated-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-first-letter-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-placeholder-visibility-1-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-placeholder-visibility-3-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-readonly-autoscroll-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-readonly-dimmed-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-readonly-empty-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-spaces-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-table-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-click-inside-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-click-outside-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-double-click-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-drag-down-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-option-delete-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-scroll-left-on-blur-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-self-emptying-click-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-text-word-wrap-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-type-text-min-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-value-expected.txt:
* LayoutTests/platform/mac/fast/forms/input-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-bidi-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-hit-test-zoomed-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-scrollbar-incremental-load-expected.txt:
* LayoutTests/platform/mac/fast/forms/listbox-width-change-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-clip-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-deselect-update-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-narrow-width-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-option-wrap-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-restrict-line-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-separator-painting-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-style-color-expected.txt:
* LayoutTests/platform/mac/fast/forms/menulist-width-change-expected.txt:
* LayoutTests/platform/mac/fast/forms/minWidthPercent-expected.txt:
* LayoutTests/platform/mac/fast/forms/number/number-appearance-rtl-expected.txt:
* LayoutTests/platform/mac/fast/forms/number/number-appearance-spinbutton-disabled-readonly-expected.txt:
* LayoutTests/platform/mac/fast/forms/number/number-appearance-spinbutton-layer-expected.txt:
* LayoutTests/platform/mac/fast/forms/option-script-expected.txt:
* LayoutTests/platform/mac/fast/forms/option-strip-whitespace-expected.txt:
* LayoutTests/platform/mac/fast/forms/option-text-clip-expected.txt:
* LayoutTests/platform/mac/fast/forms/placeholder-position-expected.txt:
* LayoutTests/platform/mac/fast/forms/placeholder-pseudo-style-expected.txt:
* LayoutTests/platform/mac/fast/forms/plaintext-mode-2-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-cancel-button-style-sharing-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-display-none-cancel-button-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-input-rtl-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-rtl-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-styled-expected.txt:
* LayoutTests/platform/mac/fast/forms/search-vertical-alignment-expected.txt:
* LayoutTests/platform/mac/fast/forms/search/search-padding-cancel-results-buttons-expected.txt:
* LayoutTests/platform/mac/fast/forms/search/search-size-with-decorations-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-background-none-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-baseline-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-block-background-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-change-listbox-size-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-change-listbox-to-popup-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-change-popup-to-listbox-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-dirty-parent-pref-widths-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-disabled-appearance-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-element-focus-ring-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-empty-option-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-initial-position-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-item-background-clip-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-list-box-with-height-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-listbox-multiple-no-focusring-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-non-native-rendering-direction-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-overflow-scroll-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-overflow-scroll-inherited-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-selected-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-style-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-visual-hebrew-expected.txt:
* LayoutTests/platform/mac/fast/forms/select-writing-direction-natural-expected.txt:
* LayoutTests/platform/mac/fast/forms/select/optgroup-rendering-expected.txt:
* LayoutTests/platform/mac/fast/forms/selectlist-minsize-expected.txt:
* LayoutTests/platform/mac/fast/forms/stuff-on-my-optgroup-expected.txt:
* LayoutTests/platform/mac/fast/forms/tabbing-input-iframe-expected.txt:
* LayoutTests/platform/mac/fast/forms/targeted-frame-submission-expected.txt:
* LayoutTests/platform/mac/fast/forms/textAreaLineHeight-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-align-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-placeholder-pseudo-style-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-placeholder-visibility-1-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-placeholder-visibility-2-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-scrollbar-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-scrolled-type-expected.txt:
* LayoutTests/platform/mac/fast/forms/textarea-setinnerhtml-expected.txt:
* LayoutTests/platform/mac/fast/forms/textfield-focus-ring-expected.txt:
* LayoutTests/platform/mac/fast/forms/textfield-outline-expected.txt:
* LayoutTests/platform/mac/fast/forms/textfield-overflow-expected.txt:
* LayoutTests/platform/mac/fast/forms/visual-hebrew-text-field-expected.txt:
* LayoutTests/platform/mac/fast/frames/take-focus-from-iframe-expected.txt:
* LayoutTests/platform/mac/fast/hidpi/resize-corner-hidpi-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/mac/fast/invalid/014-expected.txt:
* LayoutTests/platform/mac/fast/lists/dynamic-marker-crash-expected.txt:
* LayoutTests/platform/mac/fast/loader/non-html-load-event-expected.txt:
* LayoutTests/platform/mac/fast/overflow/overflow-x-y-expected.txt:
* LayoutTests/platform/mac/fast/overflow/scroll-nested-positioned-layer-in-overflow-expected.txt:
* LayoutTests/platform/mac/fast/overflow/scrollRevealButton-expected.txt:
* LayoutTests/platform/mac/fast/parser/document-write-option-expected.txt:
* LayoutTests/platform/mac/fast/parser/entity-comment-in-textarea-expected.txt:
* LayoutTests/platform/mac/fast/parser/open-comment-in-textarea-expected.txt:
* LayoutTests/platform/mac/fast/repaint/renderer-destruction-by-invalidateSelection-crash-expected.txt:
* LayoutTests/platform/mac/fast/repaint/search-field-cancel-expected.txt:
* LayoutTests/platform/mac/fast/repaint/select-option-background-color-expected.txt:
* LayoutTests/platform/mac/fast/repaint/subtree-root-skipped-expected.txt:
* LayoutTests/platform/mac/fast/replaced/replaced-breaking-expected.txt:
* LayoutTests/platform/mac/fast/replaced/replaced-breaking-mixture-expected.txt:
* LayoutTests/platform/mac/fast/replaced/three-selects-break-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-button-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-menulist-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-searchfield-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-textarea-expected.txt:
* LayoutTests/platform/mac/fast/replaced/width100percent-textfield-expected.txt:
* LayoutTests/platform/mac/fast/selectors/064-expected.txt:
* LayoutTests/platform/mac/fast/spatial-navigation/snav-multiple-select-focusring-expected.txt:
* LayoutTests/platform/mac/fast/table/003-expected.txt:
* LayoutTests/platform/mac/fast/table/append-cells2-expected.txt:
* LayoutTests/platform/mac/fast/table/colspanMinWidth-expected.txt:
* LayoutTests/platform/mac/fast/table/remove-td-display-none-expected.txt:
* LayoutTests/platform/mac/fast/table/spanOverlapRepaint-expected.txt:
* LayoutTests/platform/mac/fast/table/text-field-baseline-expected.txt:
* LayoutTests/platform/mac/fast/text/drawBidiText-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-listbox-atsui-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-listbox-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-menulist-expected.txt:
* LayoutTests/platform/mac/fast/text/international/hindi-spacing-expected.txt:
* LayoutTests/platform/mac/fast/text/international/pop-up-button-text-alignment-and-direction-expected.txt:
* LayoutTests/platform/mac/fast/text/international/rtl-white-space-pre-wrap-expected.txt:
* LayoutTests/platform/mac/fast/text/international/unicode-bidi-plaintext-in-textarea-expected.txt:
* LayoutTests/platform/mac/fast/text/textIteratorNilRenderer-expected.txt:
* LayoutTests/platform/mac/fast/transforms/transformed-focused-text-input-expected.txt:
* Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h:
* Source/WebCore/style/StyleAdjuster.cpp:
(WebCore::Style::Adjuster::adjust const):

Canonical link: https://commits.webkit.org/253424@main
